packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.spec.ts

```diff
@@ -98,7 +98,9 @@ describe('AnchorTabs', () => {
         expect(element.activetab).toBe(tab(1));
     });
 
-    it('should set aria-selected on active tab', () => {
+    // ariaSelected property is not supported on Firefox:
+    // see https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaSelected#browser_compatibility
+    it('#failsFirefox should set aria-selected on active tab', () => {
```

Comment:
We don't leverage the native attribute reflection on our components. We apply the ARIAGlobalStatesAndProperties mixin: https://github.com/microsoft/fast/blob/1be91f1b6212a55963cdaaa0cd8af76b4d7091fa/packages/web-components/fast-foundation/src/patterns/aria-global.ts

That defines the property on the prototype and the attribute synchronization. That set of attributes applies to all elements in a page.

For aria attributes specific to certain widgets with specific roles, we need to define the aria attributes associated with those roles ourselves (like aria-selected). This is probably bigger than just the anchorTabs, we need to make sure we are implementing those consistently across our controls and capture some documentation on it.

Once that is done this PR will be unnecessary. I also want to make sure that if we are disabling tests for specific browsers that we have a clear path to resolution and discuss mitigation strategies first (polyfills, changing implementation to be cross-browser, filing bugs to browsers, etc).

Reply:
I agree with that philosophy but also want to be pragmatic about moving us forward incrementally from where we are now. I'd propose a staging policy something like this:
1. add the capability to disable tests in specific browsers and document our policies around doing so. This could include filing an issue for each test (or group of tests with a common cause) and strongly discouraging new suppressions unless we've made a valiant effort to fix/polyfill them first
2. disable the existing test failures (while following this policy) in service of enabling the tests on the PR build
3. over time, prioritize fixing and re-enabling the failing tests while still holding a high bar for new regressions

I recognize there could be debate about the order of these and that disabling a bunch of tests now may result in us deprioritizing the work of fixing them. But I believe that's a better situation than we're in now, where we don't run the tests at all so we don't catch any regressions.

But perhaps it's a false choice if Mert has the bandwidth to fix these immediately!

Reply:
I'm not sure how to proceed. Should I:
1. submit the part of this change that allows us to tag tests with `#failsFirefox` to skip them?
2. add the same kind of support for skipping tests on Chrome and Webkit/Safari? (those are the only other browsers we have test commands for in package.json)
3. document our policy for skipping tests? (if so, do we need to get consensus on that policy?)
4. disable existing failing Firefox tests?
5. file a tech debt issue to review existing components for missing aria property declarations?

Reply:
Created tech debt item #1160 to audit components for missing aria attribute/property definitions.

Reply:
- Fixed 6 failing Tooltip tests, leaving 10 remaining to be skipped
- Tagged 10 tests to be skipped on Firefox
- Added Firefox testing to our `npm run test` command, thereby enabling it on the CI build
- Documented policy for skipping tests in our CONTRIBUTING doc
- Updated the test commands for Chrome and Webkit so that #failsChrome and #failsWebkit will cause tests to be skipped on those browsers

Reply:
Reminder to update the PR title and description to describe this additional work that you've done!